assets/dev/scss/editor/panel/controls/_select.scss

```diff
@@ -39,4 +39,14 @@
 			pointer-events: none;
 		}
 	}
+
+	.elementor-control-field {
+
+		&.elementor-control-field-select-small {
```

Comment:
Is the & really necessary in this case ? 
why not just write: 
```
.elementor-control-field {
  .elementor-control-field-select-small {
    .elementor-control-input-wrapper {
      max-width: 80px;
    }
  }
}
```

Reply:
Indeed, it seems to me more coherent

Reply:
Thank you for your feedback, @rodolphebertozzo .

Are you part of the Elementor team or you part of the community? What kind of work do you in the Elementor context? #curious

Reply:
I'm part of the community in a way and I'm a code contributor to improve the widgets, I'm not a developer, I learn languages by myself, especially SASS, by the way, I'll be glad if you can have a look at this PR ;) https://github.com/elementor/elementor/pull/20906

Reply:
@rodolphebertozzo I have reverse my code. My change wasn't working correctly.